_layouts/archive.hbs

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
-<html>
+<html class="flex flex-1 min-h-full">
```

Comment:
I'm pretty sure we can drop the `flex-1` on these

Reply:
Removed from `html`, this is now the only class on the `body` and `.wrapper`